bindings/rust/standard/bench/src/harness/mod.rs

```diff
@@ -399,7 +433,13 @@ mod tests {
             TlsConnPair::<C, S>::new_bench_pair(CryptoConfig::default(), HandshakeType::Resumption)
                 .unwrap();
         conn_pair.handshake().unwrap();
+        // read the session tickets which were sent
+        let err = conn_pair.client_mut().recv(&mut[0]).unwrap_err();
```

Comment:
Did you confirm this works? I have a vague memory of s2n_recv bailing early for zero-length data, and I don't remember whether or not I fixed that.